src/xo-app/home/index.js

```diff
@@ -233,6 +247,43 @@ export default class Home extends Component {
   _updateSelectedHosts = hosts => { this.setState({ selectedHosts: hosts }) }
   _updateSelectedTags = tags => { this.setState({ selectedTags: tags }) }
 
+  // Checkboxes
+  selectedVms = []
+
+  _isChecked = (id) => includes(this.selectedVms, id)
+  _updateMasterCheckbox = () => {
```

Comment:
If this function does not require to be binded (not passed as a prop in `render()`), it should be declared as a standard method.